census/README.md

```diff
@@ -1,313 +1,322 @@
+    - [TensorFlow Core](tensorflowcore/trainer/task.ipynb) (Open in [Colab](https://colab.research.google.com/github/GoogleCloudPlatform/cloudml-samples/blob/master/census/tensorflowcore/trainer/task.ipynb))
 
-* [TensorFlow Core](tensorflowcore/trainer/task.ipynb) (Open in [Colab](https://colab.research.google.com/github/GoogleCloudPlatform/cloudml-samples/blob/master/census/tensorflowcore/trainer/task.ipynb))
+#
+* **Data description**
 
-## Dataset
 The [Census Income Data
 Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) that this sample
 uses for training is hosted by the [UC Irvine Machine Learning
-Repository](https://archive.ics.uci.edu/ml/datasets/). We have hosted the data
-on Google Cloud Storage in a slightly cleaned form:
+Repository](https://archive.ics.uci.edu/ml/datasets/).
+
+Using census data which contains data a person's age, education, marital status, 
+and occupation (the features), we will try to predict whether or not the person earns 
+more than 50,000 dollars a year (the target label). We will train a logistic regression 
+model that, given an individual's information, outputs a number between 0 and 1, this can 
+be interpreted as the probability that the individual has an annual income of over 50,000 dollars.
 
- * Training file is `adult.data.csv`
- * Evaluation file is `adult.test.csv`
+As a modeler and developer, think about how this data is used and the potential benefits and harm a model's predictions can cause. A model like this could reinforce societal biases and disparities. Is each feature relevant to the problem you want to solve or will it introduce bias? For more information, read about [ML fairness](https://developers.google.com/machine-learning/fairness-overview/).
+ 
+* **Disclaimer**
 
-### Disclaimer
 This dataset is provided by a third party. Google provides no representation,
 warranty, or other guarantees about the validity or any other aspects of this dataset.
 
-### Set Environment Variables
-Please run the export and copy statements first:
+* **Setup and test your GCP environment**
 
-```
-TRAIN_FILE=gs://cloud-samples-data/ml-engine/census/data/adult.data.csv
-EVAL_FILE=gs://cloud-samples-data/ml-engine/census/data/adult.test.csv
-```
+The best way to setup your GCP project is to use this section in this
+[tutorial](https://cloud.google.com/ml-engine/docs/tensorflow/getting-started-training-prediction#set-up-your-gcp-project).
 
-### \*Optional\* Use local training files.
+* **Environment setup:**
 
-Since TensorFlow - not the Cloud ML Engine - handles reading from GCS, you can run all commands below using these environment variables. However, if your network is slow or unreliable, you may want to download the files for local training.
+Virtual environments are strongly suggested, but not required. Installing this
```

Comment:
this is very general and not census-specific.  is there a better place for this?

Reply:
Agree, my goal was that this README.md to be self-contained with mininum back and forth between different docs. I can link the reference to CMLE site for virtualenv, but though would be useful to just tell Devs, run this command and move on to next step.